server/index.js

```diff
@@ -133,15 +133,42 @@ app.delete('/api/delete/category/:id', (req, res, next) => {
               where "categoryId" = $1
               returning * `;
   const params = [deleteId];
-
   db.query(sql, params)
     .then(result => {
+
       const menuRow = result.rows;
       res.json(menuRow);
     })
     .catch(err => next(err));
 });
 
+app.put('/api/preparing/:id', (req, res, next) => {
+  const cartId = parseInt(req.params.id, 10);
+  const sql = `update "orders" set "orderStatus" =$1
+               where "cartId" = $2
+              returning * `;
+  const params = ['preparing', cartId];
+  db.query(sql, params)
+    .then(result => {
+      const statusUpdateRow = result.rows;
+      res.json(statusUpdateRow);
+    })
+    .catch(err => next(err));
+});
+app.delete('/api/ready/:id', (req, res, next) => {
+  const cartId = parseInt(req.params.id, 10);
+  const sql = `delete from "orders"
+              where "cartId" = $1
+              returning * `;
+  const params = [cartId];
+  db.query(sql, params)
+    .then(result => {
+      const deletedRow = result.rows;
+      res.json(deletedRow);
+    })
+    .catch(err => next(err));
+});
```

Comment:
Oh, you don't want to delete the order! You should mark it with a boolean column for "finished" or something like that.